requirements.txt

```diff
@@ -45,3 +48,7 @@ wget
 gym==0.24.0
 gym-super-mario-bros==7.3.0
 timm
+
+# flava tutorial - multimodal
+packaging
```

Comment:
I don't think we need `packaging` anymore. I removed it in this [PR](https://github.com/facebookresearch/multimodal/pull/322/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552).